api/app/controllers/application_controller.rb

```diff
@@ -1,4 +1,26 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  include JsonWebToken
```

Comment:
Is this the best way to use the methods on the `JsonWebToken` module, or would there be a different way that is more appropriate?

Reply:
Yes, modules are a good way to compose and share methods like this.